giskard/commands/cli_worker.py

```diff
@@ -160,9 +160,11 @@ def _start_command(is_server, url: AnyHttpUrl, api_key, is_daemon, hf_token=None
 
             run_daemon(is_server, url, api_key, hf_token)
         else:
-            if sys.platform == "win32":
+            if settings.force_asyncio_event_loop or sys.platform == "win32":
```

Comment:
Can be simplified in `elif`

Reply:
hmm, not sure since there's a common `run(...)` on line 169 that should be executed in bot "else" subbranches.

Reply:
My bad I misread the indentation